doc/examples/datasets_train/datasets_train.py

```diff
@@ -118,15 +164,48 @@ def create_data_chunk(n, d, seed, include_label=False):
         #  first N files rather than write to disk
         # to simulate a user with local copy of subset of data
 
-    # os.system("aws s3 sync ./data s3://cuj-big-data/data")
-    # os.system("aws s3 sync ./inference s3://cuj-big-data/inference")
+    if upload_to_s3:
+        os.system("aws s3 sync ./data s3://cuj-big-data/data")
+        os.system("aws s3 sync ./inference s3://cuj-big-data/inference")
```

Comment:
These are private and also not sure if `cuj-big-data` makes sense to users. Do you want to just make this configurable to the user?